app/assets/javascripts/select-kit/addon/components/dropdown-select-box/dropdown-select-box-header.js

```diff
@@ -4,10 +4,12 @@ import { readOnly } from "@ember/object/computed";
 
 export default SingleSelectHeaderComponent.extend({
   classNames: ["dropdown-select-box-header"],
-  classNameBindings: ["btnClassName", "btnStyleClass"],
+  classNameBindings: ["btnClassName", "btnStyleClass", "btnCustomClasses"],
   showFullTitle: readOnly("selectKit.options.showFullTitle"),
   customStyle: readOnly("selectKit.options.customStyle"),
 
+  btnCustomClasses: readOnly("selectKit.options.btnCustomClasses"),
```

Comment:
Not sure if this is the best way of going about things, but I couldn't see another way. I need the button to have the `btn-flat` class applied and the existing options didn't allow for this.

Reply:
Seems ok to me :+1: